scripts/build/p-sqlite-linux-ubuntu.inc

```diff
@@ -23,9 +23,13 @@ install_binary_artifact_sqlite() {
 # Check if the binary artifact is installed
 is_installed_sqlite() {
   [[ -f /usr/lib/x86_64-linux-gnu/libsqlite3.so ]] || return 1
-  return 0
+  [[ -f /usr/include/sqlite3.h ]] || return 1
```

Comment:
Was there a case where this header was missing? Also why remove `return 0`?

Reply:
Yes, missing header files can be an issue, which I happily ran into it. Ubuntu for example provides: `libsqlite3-0` for the library and ` libsqlite3-dev` for the header.

`return 0` is default.

Reply:
But you have `return 0` elsewhere